pages/mech/index.jsx

```diff
@@ -12,7 +9,7 @@ export async function getServerSideProps(ctx) {
   if (!id) {
     return {
       redirect: {
-        destination: `/mech?id=${DEFAULT_MECH_CONTRACT_ADDRESS}&hash=${DEFAULT_MECH_HASH}`,
+        destination: `/mech?id=${DEFAULT_MECH_CONTRACT_ADDRESS}`,
```

Comment:
I understand the desire to maintain the existing url format for now, so that we don't break existing links. But by default we should use the correct structure: `/mech/[address]`